### PR TITLE
Remove docs and some known broken symlinks

### PR DESCRIPTION
**Pull request by c4rt0**
Fixes: https://github.com/openshift/os/issues/1003

### CODE REVIEW

File: tests/kola/files/commonlib.sh [hunk 1]
```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
```
Comment (travier) [inline tests/kola/files/commonlib.sh:1]: Replace this one with a symlink pointing to the commonlib.s files in shared folder.

File: tests/kola/files/validate-symlinks [hunk 2]
```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# kola: { "exclusive": false, "distros": "fcos", "platforms": "qemu" }
```
Comment (travier) [inline tests/kola/files/validate-symlinks:2]:
```suggestion
# kola: { "exclusive": false, "platforms": "qemu" }
```

File: tests/kola/files/validate-symlinks [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+#   - Only run on FCOS until we've tested it on RHCOS.
+# - platforms: qemu
+#   - This test should pass everywhere if it passes anywhere.
+
+set -xeuo pipefail
+
+. commonlib.sh
+
+# List of known broken symlinks that require further investigation:
+list_broken_symlinks_skip=(
```
Comment (travier) [inline tests/kola/files/validate-symlinks:17]: Let's update this list for RHCOS

File: tests/kola/files/validate-symlinks [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+# See: https://github.com/openshift/os/issues/1003
+#
+# - distros: fcos
+#   - Only run on FCOS until we've tested it on RHCOS.
+# - platforms: qemu
+#   - This test should pass everywhere if it passes anywhere.
+
+set -xeuo pipefail
+
+. commonlib.sh
```
Comment (travier) [inline tests/kola/files/validate-symlinks:14]: Update this one to use the new symlink

Comment (c4rt0) [reply tests/kola/files/validate-symlinks]: All above are now complete.

**Comment by travier** [comment]
Let's update the title of this PR to:
```
Remove docs and some known broken symlinks
```

And the content of the first message should be:
```
Fixes: https://github.com/openshift/os/issues/1003
```

**Comment by travier** [comment]
This depends on https://github.com/openshift/os/pull/1059 for the test fix.

**Pull request merged**